jcl/src/java.base/share/classes/openj9/internal/foreign/abi/InternalDowncallHandler.java

```diff
@@ -42,55 +40,35 @@
 
 /*[IF JAVA_SPEC_VERSION >= 20]*/
 import java.lang.foreign.Arena;
-/*[ENDIF] JAVA_SPEC_VERSION >= 20 */
-/*[IF JAVA_SPEC_VERSION >= 19]*/
-/*[IF JAVA_SPEC_VERSION == 19]*/
-import java.lang.foreign.Addressable;
-/*[ENDIF] JAVA_SPEC_VERSION == 19 */
 import java.lang.foreign.FunctionDescriptor;
 import java.lang.foreign.GroupLayout;
-/*[IF JAVA_SPEC_VERSION == 19]*/
-import java.lang.foreign.MemoryAddress;
-/*[ENDIF] JAVA_SPEC_VERSION == 19 */
 import java.lang.foreign.MemoryLayout;
 import java.lang.foreign.MemorySegment;
-/*[IF JAVA_SPEC_VERSION == 19]*/
-import java.lang.foreign.MemorySession;
-/*[ENDIF] JAVA_SPEC_VERSION == 19 */
 import java.lang.foreign.SegmentAllocator;
-/*[IF JAVA_SPEC_VERSION >= 20]*/
+/*[IF JAVA_SPEC_VERSION == 20]*/
 import java.lang.foreign.SegmentScope;
-/*[ENDIF] JAVA_SPEC_VERSION >= 20 */
-import java.lang.foreign.ValueLayout;
-/*[IF JAVA_SPEC_VERSION >= 20]*/
-import static java.lang.foreign.ValueLayout.*;
-/*[ENDIF] JAVA_SPEC_VERSION >= 20 */
+/*[ENDIF] JAVA_SPEC_VERSION == 20 */
 import java.lang.foreign.VaList;
-/*[IF JAVA_SPEC_VERSION >= 20]*/
+import java.lang.foreign.ValueLayout;
 import jdk.internal.foreign.Utils;
 import jdk.internal.foreign.abi.LinkerOptions;
-/*[ENDIF] JAVA_SPEC_VERSION >= 20 */
 import jdk.internal.foreign.MemorySessionImpl;
-/*[ELSE] JAVA_SPEC_VERSION >= 19 */
+/*[ELSEIF JAVA_SPEC_VERSION == 17]*/
```

Comment:
Please simplify this (and similar places) to just `[ELSE]`:
```java
/*[ELSE] JAVA_SPEC_VERSION >= 20 */
```

Reply:
Agreed and updated everywhere against the suggestion above.